internal/cli/auth/login.go

```diff
@@ -88,6 +88,16 @@ func (opts *loginOpts) SetOAuthUpAccess() {
 }
 
 func (opts *loginOpts) Run(ctx context.Context) error {
+	// Check if user has already logged in
+	if s, err := opts.config.AccessTokenSubject(); err == nil && s != "" {
```

Comment:
what will happen when the token expires?

Reply:
users will have to first run `auth logout` and then run `auth login` as I think they are doing right now

Reply:
I see how it makes sense to check if the token is valid

Reply:
right now I don't have to login, I just re-login

Reply:
my recommendation to discuss with @dianchenghu and @boooczek is to ask users if they want to reauthenticate,
```bash
You are already authenticated as <>: do you want to re authenticate(Y/n): _
```

Reply:
I understand what we're discussing here is case when token expieres. Then when user types in "atlas login" the flow is as usual, i.e. they go through auth process in the browser.
@gssbzn why users would want to re-authenticate?

Reply:
> why users would want to re-authenticate? Is it because their token might expire soon?
- I may have multiple atlas account (I do, personal and professional) and I may want to switch between them
- See my general comment on token expiration:  tokens expire which means users need to re-authenticate, you can check if the token has expired, we have logic for that but then we have a refresh token which sadly lacks expiration embedded and you can only verify it expired by trying to use it, there are two tokens one we know when it expires the second one we don't until we try to use it, we can't change how that second token works